sys/ps/ps.c

```diff
@@ -77,18 +78,34 @@ void ps(void)
             overall_stacksz += stacksz;
             stacksz -= thread_measure_stack_free(p->stack_start);
             overall_used += stacksz;
+            int isr_usage = thread_arch_isr_stack_usage();                         /* ISR stack usage */
 #endif
 #ifdef MODULE_SCHEDSTATISTICS
             double runtime_ticks =  sched_pidlist[i].runtime_ticks / (double) xtimer_now() * 100;
             int switches = sched_pidlist[i].schedules;
 #endif
+
+#ifdef DEVELHELP
+        if(i==1){
+            printf("\t%3" PRIkernel_pid
+                   " | %-19s "
+                   " | %.8s %.8s | %.3s "
+                   " | %5i (%5i) | %.2s "
+                   "\n",
+                   0,
```

Comment:
I would prefer to simply print `-` here instead of `0`, as the isr_stack is not a thread and thereby has not PID...